### ISSUE TITLE
Latest version of plug-in/VLC not working

### ISSUE THREAD

**Issue by M4CC** [exebetche/vlsub#152]
I am running VLC 2.2.4 with VLSub 0.10.0 and I still can't download any subtitles. Tried searching by hash and by name, I always get a message to kill the plug-in and when I do so it force quits VLC. How can I fix this? Thanks!

**Comment by FredyC** [comment]
I have a same problem for like last 14 days or so. I am not sure what's wrong so suddenly, it was working just fine before. I click on either of two buttons (by hash or by name) and nothing is happening. The window is not frozen, it's still possible to click around and type, but it's not possible to click on "Close" button and after couple seconds it wants to be killed.

> Windows 10 x64, VLC 2.2.4, VLsub 0.9.13

**Comment by M4CC** [comment]
Yeah, I already reported it on: https://addons.videolan.org/content/show.php/VLSub+0.9.13?content=148752 and tried to find a solution among the comments but nothing. Giving it a shot here now.

**Comment by FredyC** [comment]
Actually I just updated the latest vlsub from master branch here which is 0.10 and that seems to be working, at least so far. Give it a try @M4CC

**Comment by M4CC** [comment]
Still get stuck at "Logging in" when I try searching for a subtitle using either method.

**Comment by MartenBE** [comment]
I got the same problem, it got stuck on calculating the hash every time. manually reinstalling the lua file from here in my vlc application (version 2.2.4) fixed it.

**Comment by f-ckth-ssh-t** [comment]
I did what Marten did but to no avail.

LIke another user I can't search any subtitle. It says "Error: code '401 Unauthorized' (200)", while opensubtitles.org seems to be online

Sorry for the crosspost: https://github.com/exebetche/vlsub/issues/96

**Comment by M4CC** [comment]
Me too... I downloaded this version but the problem still persists. Is there actually any difference between this GitHub version and the one posted on the main plug-in's website?

**Comment by MartenBE** [comment]
In my case, the hash function and search by name function now works as it should, but when downloading/loading the subtitles it still hangs for 5 seconds. I thought this had something to do with the connection to the website. It is usable though.

**Comment by f-ckth-ssh-t** [comment]
The versions differ: from within VLC it shows 0.9.8 but when I download from Github it says 0.10.0.

Version 0.9.8 stays searching and does nothing.
Version 0.10 gives the error 401 like mentioned.

**Comment by naoufal450** [comment]
I tried this and it worked just fine.
Go to the extention's configuration and click on "VLSub working directory" and delete the file "vlsub_conf.xml".
I hope this will help.

**Comment by f-ckth-ssh-t** [comment]
I tried it but it doesn't work. Which version of VLsub are you using?

**Comment by naoufal450** [comment]
i use 0,10
i had the same problem can't download but it shows search results...i tried this and it works.(maybe because the configuration file is not suited for this version idk).

make you sure you didn't edit your host file

**Comment by f-ckth-ssh-t** [comment]
Thanks but mine does not show search results.

I think I changed by host file back again correctly but I am not 100%
sure.  Is it a good idea to reinstall VLC then?

**Comment by nabilfreeman** [comment]
VLSub 0.10 is working for me with Mac OS X 10.11.6 and VLC 2.2.4. I followed the instructions in #149 to update from the old 0.9.13 version 💪

**Comment by thevtm** [comment]
When I search for a subtitle I get the "Extension 'VLSub 0.9.13' does not respond ..." after a 1-2 minutes it start responding again. This also happens when I click on the close button.
It's happening to me in 2 different computers and 3 different OSes (Win10 / Win7 / Arch Linux).

VLC 2.2.4 -> VLSub 0.9.13

**Comment by pgcalixto** [comment]
@thevtm, this also happens to me. I'm using it in Brazil, with the same VLC and plugin versions, and it shows the same message, language-independently.

**Comment by varjolintu** [comment]
Here's a quick and dirty fix for the current version (from HTTP 1.1 to HTTP 1.0):
https://dl.dropboxusercontent.com/u/55238633/vlsub.lua

**Comment by pgcalixto** [comment]
Awesome! This solved my problem :D @varjolintu

**Comment by meltdown03** [comment]
Thanks @varjolintu!

**Comment by varjolintu** [comment]
It seems that the official version has been updated with the same changes I made. However, the correct fix should be in the chunked transfer downloading section. Switching to HTTP 1.0 is not a permanent solution.

**Comment by varjolintu** [comment]
I've sent a pull request that fixes the problem with HTTP 1.1. Let's hope it will be included to the main branch soon. More info here:
https://github.com/varjolintu/vlsub/tree/download_fix

**Comment by TonyWhitley** [comment]
Using 0.10.0 I get the message that it does not respond but if I ignore that vlsub will download the subtitles a little later.  It seems that VLC is just timing out too quickly but I could not find an option to change it.

**Comment by Mancerrss** [comment]
I hope you have an auto updater mechanism, instead of relaying only in newer VLC updates, which they're far too slow now and far too late on their update plan

**Comment by tiagocardosoweb** [comment]
Here it's not working since a longe time! Updated to 0.10.2 and still.

**Comment by lguerard** [comment]
Same here. VLsub 0.10.2 and 0.9 saying code '401 Unauthorized' (200).

Any ideas on what to do ? Thanks !

**Comment by TonyWhitley** [comment]
I can't be much help other than to say 0.10.2 is working fine for me.

**Comment by asad9711** [comment]
download this lua file [https://raw.githubusercontent.com/exebetche/vlsub/master/vlsub.lua](url) and place in the directory as mentioned here [https://addons.videolan.org/p/1154045/](url). It works in my case. I'm using VLC 2.2.4

**Comment by ecsekitibi** [comment]
got the 401 error,
registered on OS.org
tired to login on VLSUB
VLC frozen. Still stuck. Any working method? vlc3.0.8/vlsub0.11.0/Mojave10.14.6
ps. to dev: .Why do you need to fuck up stg what has worked flawless for years?
